app/src/main/java/ch/epfl/sdp/blindly/settings/Settings.kt

```diff
@@ -0,0 +1,92 @@
+package ch.epfl.sdp.blindly.settings
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.SeekBar
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import ch.epfl.sdp.blindly.R
+
+
+const val EXTRA_LOCATION = "user_location"
+const val EXTRA_SHOW_ME = "user_show_me"
+//const val REQUEST_LOCATION = 1

```

Comment:
Is this comment still needed, also are these used outside, maybe they could be private ?

Reply:
They're used by the children Activities, and the commented one will be used next week

Reply:
All good, thanks!